### PR TITLE
 ENH Add config to control if a default author is added to posts on creation

### PR DESCRIPTION
**Pull request by jellygnite**

### CODE REVIEW

**Comment by GuySartorelli** [comment]
Note: there is some discussion about this in #668

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 /node_modules/
 **/*.js.map
 **/*.css.map
+/.vs
```
Comment (dhensby) [inline .gitignore:6]: please don't add IDE specific ignores - you can add these to your own environment globally. see https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer

**Review by dhensby — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (jellygnite) [reply .gitignore]: sorry. I knew that would come back when I saw it as part of the commit. Should be resolved now.

**Comment by dhensby** [comment]
How do you feel about adding a test to cover this new behaviour?

**Review by dhensby — APPROVED** [review]
I'll approve it as is, but leave it to another contributor to agree and merge

**Review by michalkleiner — APPROVED** [review]

**Comment by michalkleiner** [comment]
Created a new enhancement issue for adding the tests — https://github.com/silverstripe/silverstripe-blog/issues/670.

**Comment by michalkleiner** [comment]
@jellygnite thank you for dipping your toes in the PR waters and submitting your contribution, we do appreciate each and every one of them.

With the tests, there's a new issue for it (linked above), so when (and if) you feel ready, give it a go. In the linked test file there are already existing tests for other aspects of the BlogPost class, so you can take some hints/inspiration from there. Definitely give it a try, submit your change as a new PR, and when a maintainer here has a bit of time, they can help you push it over the line.

**Pull request merged**